src/components/molecules/article-status/article-status.tsx

```diff
@@ -49,13 +50,22 @@ export const ArticleStatus = ({
       <Modal modalTitle={'Share this article'} open={showShareModal} onModalClose={() => setShowShareModal(false)} modalLayout="share">
         <div className="form-item">
           <input readOnly={true} type="input" className="text-field text-field--clipboard" value={`https://doi.org/${doi}`} />
-          <Clipboard text={`https://doi.org/${doi}`} />
+          <div className="clipboard-container">
```

Comment:
Why has this div been moved outside of the clipboard.tsx?

Reply:
Because it was affecting the positioning of the button within the modal, which made it a pain for us to use it in a group of buttons